dali/kernels/erase/erase_gpu.h

```diff
@@ -376,10 +378,9 @@ struct EraseGpu {
   using sample_t = erase_sample_desc<T, ndim>;
 
   KernelRequirements Setup(KernelContext &context,
-                          //  const TensorListView<StorageGPU, const T, ndim> &in
                            const InListGPU<T, ndim> &in,
-                           const InListGPU<ibox<ndim>, 1> &erased_regions,
-                           span<const T> fill_values = {}) {
+                           const std::vector<std::vector<ibox<ndim>>> &erased_regions,
+                           const span<const T> fill_values = {}) {
```

Comment:
This is unnecessary - constness of span contents is determined by the element type, whereas the span itself is passed by value.

Reply:
done